sys/include/xtimer.h

```diff
@@ -184,120 +189,129 @@ static inline void xtimer_tsleep64(xtimer_ticks64_t ticks);
  *
- * This function sets a timer that will send a message @p offset microseconds
- * from now.
+ * This function sets a timer that will send a message @p offset_us
+ * microseconds from now.
  *
- * The mesage struct specified by msg parameter will not be copied, e.g., it
+ * The message struct specified by msg parameter will not be copied, e.g., it
  * needs to point to valid memory until the message has been delivered.
  *
- * @param[in] timer         timer struct to work with.
- *                          Its xtimer_t::target and xtimer_t::long_target
- *                          fields need to be initialized with 0 on first use.
- * @param[in] offset        microseconds from now
- * @param[in] msg           ptr to msg that will be sent
- * @param[in] target_pid    pid the message will be sent to
+ * @param[in] timer        timer struct to work with.
+ *                         Its xtimer_t::target and xtimer_t::long_target
+ *                         fields need to be initialized with 0 on first use.
+ * @param[in] offset_us    microseconds from now
+ * @param[in] msg          ptr to msg that will be sent
+ * @param[in] target_pid   pid the message will be sent to
  */
-static inline void xtimer_set_msg64(xtimer_t *timer, uint64_t offset, msg_t *msg, kernel_pid_t target_pid);
+static inline void xtimer_set_msg64(xtimer_t *timer, uint64_t offset_us,
+                                    msg_t *msg, kernel_pid_t target_pid);
 
 /**
  * @brief Set a timer that wakes up a thread
  *
- * This function sets a timer that will wake up a thread when the timer has
- * expired.
+ * This function sets a timer that will call thread_wakeup on the given @p pid
+ * after @p offset_us microseconds have passed. The thread with @p pid is not
+ * set to sleep.
  *
- * @param[in] timer         timer struct to work with.
- *                          Its xtimer_t::target and xtimer_t::long_target
- *                          fields need to be initialized with 0 on first use
- * @param[in] offset        microseconds from now
- * @param[in] pid           pid of the thread that will be woken up
+ * @param[in] timer       timer struct to work with.
+ *                        Its xtimer_t::target and xtimer_t::long_target
+ *                        fields need to be initialized with 0 on first use
+ * @param[in] offset_us   microseconds from now
+ * @param[in] pid         pid of the thread that will be woken up
  */
-static inline void xtimer_set_wakeup(xtimer_t *timer, uint32_t offset, kernel_pid_t pid);
+static inline void xtimer_set_wakeup(xtimer_t *timer, uint32_t offset_us,
+                                     kernel_pid_t pid);
 
 /**
  * @brief Set a timer that wakes up a thread, 64bit version
  *
- * This function sets a timer that will wake up a thread when the timer has
- * expired.
+ * This function sets a timer that will call thread_wakeup on the given @p pid
+ * after @p offset_us microseconds have passed. The thread with @p pid is not
+ * set to sleep.
```

Comment:
The second sentence could be generalized to "This function does not put any threads to sleep"